### DISCUSSION TITLE
AzViz doesn't seem to find graphviz

### DISCUSSION THREAD

**Discussion by securitymonster** [PrateekKumarSingh/AzViz#121] [General]
I'm on Apple Silicon. I installed Powershell and graphviz using homebrew. In powershell AzViz installs fine as well. However when I run Export-AzViz I get the message:

'GraphViz' is not installed on this system and is a prerequisites for this module to work.

However it is installed. I can type dot -v in powershell and it will output the version etc., see below.  Any idea's how to fix this?

```
PS /opt/local/bin> dot -v
dot - graphviz version 9.0.0 (20230911.1827)
libdir = "/opt/homebrew/Cellar/graphviz/9.0.0/lib/graphviz"
Activated plugin library: libgvplugin_dot_layout.6.dylib
Using layout: dot:dot_layout
Activated plugin library: libgvplugin_core.6.dylib
Using render: dot:core
Using device: dot:dot:core
The plugin configuration file:
	/opt/homebrew/Cellar/graphviz/9.0.0/lib/graphviz/config6
		was successfully loaded.
```

**Comment by securitymonster** [comment]
Ok I found the issue. I had a look at the code and found that Get-DOTExecutable.ps1 doesn't include the location for homebrew's location of graphviz. So I added the homebrew path as follows:

```
function Get-DOTExecutable {

    $PossibleGraphVizPaths = @(
        'C:\Program Files\NuGet\Packages\Graphviz*\dot.exe',
        'C:\program files*\GraphViz*\bin\dot.exe',
        '/usr/local/bin/dot',
        '/usr/bin/dot',
        '/opt/homebrew/bin/dot'
    )

    $GraphViz = Resolve-Path -path $PossibleGraphVizPaths -ErrorAction SilentlyContinue | Get-Item | Where-Object BaseName -eq 'dot' | Select-Object -First 1

    return $GraphViz
}
```

**Reply by securitymonster** [reply]
I created a pull request [here](https://github.com/PrateekKumarSingh/AzViz/pull/122)

**Comment by kidchenko** [comment]
Can we approve this? 

Same issue here
